examples/kubeflow.yaml

```diff
@@ -40,7 +40,7 @@ checks:
     description: 🌏 Exposes an HTTP interface on port 8888
     probe:
       httpGet:
-        path: /
+        path: /hub/jovyan
```

Comment:
I'm curious about the motivation here. This feels orthogonal to the rest of this PR.

It's my understanding that Kubeflow only expects a service on port `8888`, not any particular path on that web server. You could equally run VSCode Server or some other application and expect it to work.